packages/HttpTransport/src/HttpTransport.js

```diff
@@ -0,0 +1,58 @@
+import get from 'lodash/get';
+import last from 'lodash/last';
+import Promise from 'bluebird';
+import { EventEmitter } from 'events';
+
+function getRequestId(command) {
+  return get(command, 'headers.id');
+}
+
+function getQueueName(command) {
+  return get(command, 'headers.queueName');
+}
+
+class HttpTransport {
+  constructor({server, log, sendingOnly}) {
+    this.log = log;
+    this.sendingOnly = sendingOnly;
+    this.server = server;
+    this.messageHandler = new EventEmitter();
+    this.requestHandler = new EventEmitter();
+
+    this.listenOnServer();
+  }
+
+  listenOnServer() {
+    console.log(HttpTransport.ENQUEUE_URI)
+    this.server.post(HttpTransport.ENQUEUE_URI, (req, res) => {
+      const command = get(req, 'body.data');
+      this.requestHandler.once(getRequestId(command), (command) => {
+        res.send(command);
+      });
```

Comment:
you need a general request handler for other command types (not just responses which is what this one is for)